src/Core/Content/Flow/Dispatching/FlowExecutor.php

```diff
@@ -200,6 +204,12 @@ private function callHandle(ActionSequence $sequence, StorableFlow $event): void
             return;
         }
 
+        if ($this->messageBus !== null && $action instanceof AsyncFlowAction) {
```

Comment:
This would mean all sendMailAction are now handled via queue?

I guess this will crash a lot of shops because in the queue you have no more request stack and i would guess a lot of plugins and implementations are accessing during mail generation.

Therefore i would recommend major flag or at least config with opt-in (next major opt-out, or remove config)